File: model/audits.go
// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
// See LICENSE.txt for license information.

package model

type Audits []Audit

func (o Audits) Etag() string {
	if len(o) > 0 {
		// the first in the list is always the most current
		return Etag(o[0].CreateAt)
	}
	return ""
}

File: webapp/platform/types/src/audits.ts
// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
// See LICENSE.txt for license information.

export type Audit = {
    id: string;
    create_at: number;
    user_id: string;
    action: string;
    extra_info: string;
    ip_address: string;
    session_id: string;
}
